manual/src/refman/extensions/extensiblevariants.etex

```diff
@@ -117,3 +117,24 @@ end = struct
   end
 end
 \end{caml_example*}
+
+Extensible variant types can be reexported in another module,
+for instance to make the type appear in the most natural module
+for the end user of a library.
```

Comment:
The paragraph and following example should probably be moved after the paragraph about rebinding a constructor, since it answers a similar concern of re-exporting objects in the right place.